src/algebra/lie/nilpotent.lean

```diff
@@ -41,6 +41,19 @@ def lower_central_series (k : ℕ) : lie_submodule R L M := (λ I, ⁅(⊤ : lie
   lower_central_series R L M (k + 1) = ⁅(⊤ : lie_ideal R L), lower_central_series R L M k⁆ :=
 function.iterate_succ_apply' (λ I, ⁅(⊤ : lie_ideal R L), I⁆) k ⊤
 
+lemma lower_central_series_antitone {k l : ℕ} (h : l ≤ k) :
```

Comment:
Does it make sense to phrase in terms of `antitone`?

Reply:
Yes and done!